docs/New Game.md

```diff
@@ -152,3 +152,28 @@ misconfiguration in Randovania.
 Before a game can work in Multiworld, it must first work properly as a solo game. It is highly recommended to finish the basic integration first.
 
 **TODO**
+
+# Expectations of a Development Game
+
+After a game has been merged and is available in development builds, the responsibility of maintaining it is now shared between you and the Randovania team:
+- Randovania: We make sure your game is included in all Randovania changes.
+- You: Work towards the game reaching stable.
+
+We're happy to support new games by doing our part, but since that has a cost, that we don't want to be doing forever, there are some expectations regarding time frames:
```

Comment:
```suggestion
We're happy to support the addition of new games by sharing what we know,  but too much investment pulls our time from core projects so we have some expectations regarding time frames:
```
The language originally used seems a bit cold and needed some polish. Open to tweaking this further.